plenum/test/node_request/test_commit/test_num_of_commit_with_f_plus_one_faults.py

```diff
@@ -18,13 +18,14 @@
 
 @pytest.fixture(scope="module")
 def setup(startedNodes):
-    A = startedNodes.Alpha
-    B = startedNodes.Beta
-    G = startedNodes.Gamma
+    A = startedNodes.Eta
```

Comment:
Do we use Nodes Eta, Gamma, Zeta to make sure that they will not become Primary (since we use round-robin, at least 2 view changes are needed to make Eta a primary)?
If so, please put a comment in the test. 
If not, please explain why we do it.